### ISSUE TITLE
BeautifulSoup warning

### ISSUE THREAD

**Issue by jaraco** [jseutter/ofxparse#86]
Late versions of BeautifulSoup issue this warning:

```
c:\program files\python 3.5\lib\site-packages\bs4\__init__.py:166: UserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("html.parser"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

To get rid of this warning, change this:

 BeautifulSoup([your markup])

to this:

 BeautifulSoup([your markup], "html.parser")

  markup_type=markup_type))
```

**Comment by jaraco** [comment]
I should note this warning comes with the 0.14 release.

**Comment by jaraco** [comment]
The warning seems to incidentally have been addressed in d1fac8cfe40d, though that change hasn't been released.

**Comment by jseutter** [comment]
Cool, thanks for the report :)
